src/odemis/acq/test/move_test.py

```diff
@@ -307,185 +315,187 @@ def setUpClass(cls):
         f.result()
-        current_grid = getCurrentGridLabel(self.stage.position.value, self.stage)
+        current_grid = self.posture_manager.getCurrentGridLabel()
         self.assertEqual(GRID_2, current_grid)
         # move back to the grid1 
-        f = cryoSwitchSamplePosition(GRID_1)
+        f = self.posture_manager.cryoSwitchSamplePosition(GRID_1)
         f.result()
-        current_grid = getCurrentGridLabel(self.stage.position.value, self.stage)
+        current_grid = self.posture_manager.getCurrentGridLabel()
         self.assertEqual(GRID_1, current_grid)
         # make sure we are still in fm imaging area
-        current_imaging_mode = getCurrentPositionLabel(self.stage.position.value, self.stage)
+        current_imaging_mode = self.posture_manager.getCurrentPostureLabel(self.stage.position.value)
         self.assertEqual(FM_IMAGING, current_imaging_mode)
 
     def test_moving_to_sem_from_fm(self):
-        f = cryoSwitchSamplePosition(LOADING)
+        f = self.posture_manager.cryoSwitchSamplePosition(LOADING)
         f.result()
         # move to the fm imaging area
-        f = cryoSwitchSamplePosition(FM_IMAGING)
+        f = self.posture_manager.cryoSwitchSamplePosition(FM_IMAGING)
         f.result()
-        current_imaging_mode = getCurrentPositionLabel(self.stage.position.value, self.stage)
+        current_imaging_mode = self.posture_manager.getCurrentPostureLabel(self.stage.position.value)
         self.assertEqual(FM_IMAGING, current_imaging_mode)
         # move to sem
-        f = cryoSwitchSamplePosition(SEM_IMAGING)
+        f = self.posture_manager.cryoSwitchSamplePosition(SEM_IMAGING)
         f.result()
-        current_imaging_mode = getCurrentPositionLabel(self.stage.position.value, self.stage)
+        current_imaging_mode = self.posture_manager.getCurrentPostureLabel(self.stage.position.value)
         self.assertEqual(SEM_IMAGING, current_imaging_mode)
 
     def test_unknown_label_at_initialization(self):
         arbitrary_position = {"x": 0.0, "y": 0.0, "z":-3.0e-3}
         self.stage.moveAbs(arbitrary_position).result()
-        current_imaging_mode = getCurrentPositionLabel(self.stage.position.value, self.stage)
+        current_imaging_mode = self.posture_manager.getCurrentPostureLabel(self.stage.position.value)
         self.assertEqual(UNKNOWN, current_imaging_mode)
-        current_grid = getCurrentGridLabel(self.stage.position.value, self.stage)
+        current_grid = self.posture_manager.getCurrentGridLabel()
         self.assertEqual(current_grid, None)
 
 
 class TestMimasMove(unittest.TestCase):
     """
-    Test move functions on the MIMAS
+    Test the MimasPostureManager functions
     """
     backend_was_running = False
 
     @classmethod
     def setUpClass(cls):
         testing.start_backend(MIMAS_CONFIG)
+        cls.microscope = model.getMicroscope()
+        cls.posture_manager = MicroscopePostureManager(microscope=cls.microscope)
```

Comment:
For all these `Test*Move` classes, I'd suggest to add a check that the posture manager is the correct type. Something like `cls.assertIsInstance(cls.posture_manager, MimasPostureManager)`.